Doorsfan/ClearBnB-Database-Group4
# Messaging

Support chat functionality. Some of this is a little clunky, but it does the job. You need to re-import to the SQL file in order to test this since I added a recipient_id column to the Message table.

To test:
1. Log in as random user
2. On bottom of home page, click on "Chat with Support" and then "connect"
3. In another window (incognito?), log in as a user with the name "support"
4. Click on the name of the user that you would like to chat with
5. Click "connect" and start sending messages
6. Note that if you refresh the page (as support) the previous messages will still be there. This is not implemented for a normal user, though.
